avaframe/com1DFA/com1DFA.py

```diff
@@ -1621,11 +1679,27 @@ def initializeResistance(cfg, dem, simTypeActual, resLine, reportAreaInfo, thres
         # Combine constants (d, cw, sres) to one parameter cRes
         cResRaster = cRes * mask
         reportAreaInfo["resistance"] = "Yes"
+
+        if DetBool:
+            resistanceArea = resLine["fileName"]
+            log.info("Initializing detrainment (resistance) area: %s" % (resistanceArea))
+            log.info("Detrainment (Resistance) area features: %s" % (resLine["Name"]))
+            resLine = geoTrans.prepareArea(resLine, dem, thresholdPointInPoly)
+            mask = resLine["rasterData"]
```

Comment:
this is already happening outside of the if -why did add this code put it here again?

aren't just the following two lines needed?